src/components/templates/ReactionPage/ReactionList.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useMemo, useState } from "react";
+import classNames from "classnames";
+
+import { DEFAULT_PARTY_NAME, DEFAULT_PROFILE_IMAGE } from "settings";
+
+import {
+  chatMessageAsMessageToTheBand,
+  Reaction,
+  ReactionsTextMap,
+} from "utils/reactions";
+import { withId, WithId } from "utils/id";
+
+import { User } from "types/User";
+import { ChatMessage } from "types/chat";
+
+import { useWorldUsersByIdWorkaround } from "hooks/users";
+
+import UserProfileModal from "components/organisms/UserProfileModal";
+import UserProfilePicture from "components/molecules/UserProfilePicture";
+import { UserAvatar } from "components/atoms/UserAvatar";
+
+export interface ReactionListProps {
+  reactions: Reaction[];
+  chats: ChatMessage[];
+  small?: boolean;
```

Comment:
?should we refactor this into `size`?

Reply:
If we did, how would you envisage it looking? Presumably wouldn’t be a Boolean anymore if we did that?

Reply:
I think since there’s only 1 option with it, it’s probably fine as is.

Reply:
`If we did, how would you envisage it looking? Presumably wouldn’t be a Boolean anymore if we did that?`

size would be `ReactionSize`. I didn't dig into the logic, but I guess it only is used in styles, so dynamic classNames would do the trick here

Reply:
Right, so an enum then. Yeah, if we had more than one option, that would definitely be the way to go, but since we don't, I don't think it matters a whole lot here. Feel free to unresolve this if you feel otherwise.